framework/src/loops/ComputeElemAuxBcsThread.C

```diff
@@ -91,6 +93,13 @@ ComputeElemAuxBcsThread<AuxKernelType>::operator()(const ConstBndElemRange & ran
           _problem.reinitMaterialsFace(elem->subdomain_id(), _tid);
 
           _problem.reinitMaterialsBoundary(boundary_id, _tid);
+
+          if (neighbor && neighbor->active())
```

Comment:
shouldn't these two cehcks be nested?,
like 

```
if (neighbor != nullptr )
  if(neighbor->active())
```

Reply:
No because if `neighbor == nullptr`, then the second boolean expression is not evaluated

Reply:
> No because if `neighbor == nullptr`, then the second boolean expression is not evaluated

Then ok, just fix the test.